sdk/python/lib/pulumi/provider/experimental/ComponentProvider.md

```diff
@@ -10,8 +10,12 @@ First we need to let Pulumi's plugin system know which language runtime should b
 
 ```yaml
 runtime: python
+name: my-provider
 ```
 
+> [!NOTE]
+> The name specified must exactly match the package name in the type string of your component.
```

Comment:
If this has to match why are we not just inferring it when we get the component type names?